{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"text":"networkError","kind":"identifier"}],"languages":["swift"],"platforms":["macOS"]}]}],"schemaVersion":{"minor":3,"patch":0,"major":0},"kind":"symbol","abstract":[{"type":"text","text":"An error occurred while reaching Firebase servers."}],"variants":[{"paths":["\/documentation\/rxfireauth\/usererror\/networkerror"],"traits":[{"interfaceLanguage":"swift"}]}],"metadata":{"symbolKind":"case","modules":[{"name":"RxFireAuth"}],"roleHeading":"Case","fragments":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"text":"networkError","kind":"identifier"}],"role":"symbol","externalID":"s:10RxFireAuth9UserErrorO07networkE0yA2CmF","title":"UserError.networkError"},"sections":[],"hierarchy":{"paths":[["doc:\/\/RxFireAuth\/documentation\/RxFireAuth","doc:\/\/RxFireAuth\/documentation\/RxFireAuth\/UserError"]]},"identifier":{"url":"doc:\/\/RxFireAuth\/documentation\/RxFireAuth\/UserError\/networkError","interfaceLanguage":"swift"},"references":{"doc://RxFireAuth/documentation/RxFireAuth/UserError/networkError":{"url":"\/documentation\/rxfireauth\/usererror\/networkerror","title":"UserError.networkError","type":"topic","abstract":[{"text":"An error occurred while reaching Firebase servers.","type":"text"}],"identifier":"doc:\/\/RxFireAuth\/documentation\/RxFireAuth\/UserError\/networkError","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"networkError","kind":"identifier"}],"kind":"symbol","role":"symbol"},"doc://RxFireAuth/documentation/RxFireAuth":{"url":"\/documentation\/rxfireauth","abstract":[],"identifier":"doc:\/\/RxFireAuth\/documentation\/RxFireAuth","role":"collection","kind":"symbol","title":"RxFireAuth","type":"topic"},"doc://RxFireAuth/documentation/RxFireAuth/UserError":{"abstract":[{"text":"Errors thrown by  ","type":"text"},{"code":"UserManagerType","type":"codeVoice"},{"type":"text","text":" implementations."}],"url":"\/documentation\/rxfireauth\/usererror","type":"topic","fragments":[{"kind":"keyword","text":"enum"},{"text":" ","kind":"text"},{"text":"UserError","kind":"identifier"}],"role":"symbol","title":"UserError","kind":"symbol","identifier":"doc:\/\/RxFireAuth\/documentation\/RxFireAuth\/UserError","navigatorTitle":[{"text":"UserError","kind":"identifier"}]}}}